src/page-sentencing/PageSentencing.js

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import DetailPage from "../detail-page";
-import sentenceTypes from "../assets/test_data/US_ND_sentence_type_by_district_by_demographics.json";
 import judicialDistricts from "../assets/test_data/US_ND_judicial_districts.json";
+import sentenceLengths from "../assets/test_data/US_ND_sentence_lengths_by_district_by_demographics.json";
```

Comment:
Just in case you're wondering, I don't care which order our two PRs are merged in. Happy to handle with migrating your stuff into the new form of my PR if necessary.